src/mixed-line-bar-chart/chart-container.tsx

```diff
@@ -391,11 +392,7 @@ export default function ChartContainer<T extends ChartDataTypes>({
 
   const onApplicationBlur = (event: React.FocusEvent<Element>) => {
     const blurTarget = event.relatedTarget || event.target;
-    if (
-      blurTarget === null ||
-      !(blurTarget instanceof Element) ||
```

Comment:
The `blurTarget instanceof Element` fails if the target belongs to a different document. As result, this if condition always resolves to true and thus causes the popover to be dismissed unexpectedly.

See:

https://github.com/user-attachments/assets/af34eeaa-95ff-49c8-b9c7-86b14330db1b

Replacing the check with `!isElement(blurTarget)` fixes the problem.